WEB_SERVICE/WEB/_MEDIUM/GRAPHQL_OBJECT_LIMIT_OVERRIDING/meta.json

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "Object Limit Overriding in GraphQL",
+  "short_description": "Vulnerability in GraphQL where attackers can exploit insufficient query limits to cause DoS or excessive resource consumption.",
+  "risk_rating": "medium",
+  "references": {
+    "GraphQL Vulnerabilities and Common Attacks: What You Need to Know": "https://www.imperva.com/blog/graphql-vulnerabilities-common-attacks/#pagination-limit-bypass",
+    "OWASP GraphQL Security Cheat Sheet": "https://cheatsheetseries.owasp.org/cheatsheets/GraphQL_Cheat_Sheet.html#batching-attacks"
+  },
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "CWE_TOP_25": [
+      "CWE_400"
```

Comment:
I don't think that `CWE_400` is  part of `CWE_TOP_25`

Reply:
It's a part of it 
![image](https://github.com/user-attachments/assets/fe7eb1c0-138d-4690-8716-f623e7fca43c)